### ISSUE TITLE
iOS 12 support requested 

### ISSUE THREAD

**Issue by badger200** [julioverne/NtSpeed#14]
Hi I love this tweak so much, I can hardly live without it. I’ve donated to you before too and I would do so again if you can fix this for 12.4. 

It installs but doesn’t display anywhere, and the Preferences bundle says “There was an error loading this preference bundle”. 

I did look at the source code and it seems super simple, even though I’m not really a developer, but I don’t know what would cause it to cease working to be honest.

**Comment by JK98062** [comment]
@badger200
Have you tried using the Network Activity StatusBar/FloatWindow on DataMeter? It does virtually the same thing but it was just developed later than NtSpeed and has a couple more features bundled in with it.

DataMeter 1.4-3 is on BigBoss repo if you wanted to try it out. Doesn't officially support iOS 12, but have seen mixed reports of it working on 12 with RootlessJB so maybe give it a go?

Also, this might be worth a read if non-functional on your iOS 12 Device: https://amp.reddit.com/r/jailbreak/comments/apvrnk/releasetweaks_datameter12fixwork_properly_in/

**Comment by badger200** [comment]
@JK98062 Actually I got NtSpeed working perfectly now, I think it was just my device’s fault - I updated from iOS 10.2 > 12.4 and didn’t uninstall any of my previous jailbreak stuff (I definitely should have in retrospect, especially since the update broke 32 bit support, which made several tweaks difficult to remove). 

I used JTool to inspect which .dylibs NtSpeed links to, but didn’t find anything relevant. Nevertheless, it didn’t work until I had manually reinstalled several support library tweaks which got broken during the iOS update. I can’t say which one solved it, but honestly Cephei was the one that brought most of my tweaks back, even though NtSpeed doesn’t seem to use it.  

I already have production App Store apps that do Notification Center widgets for network transfer data, but your FloatWindow does sound interesting. I will check it out, thanks!
